pkg/kudoctl/cmd/install/install.go

```diff
@@ -100,9 +100,14 @@ func installFrameworks(args []string, options *Options) error {
 func getPackageCRDs(name string, options *Options, repository repo.FrameworkRepository) (*repo.PackageCRDs, error) {
 	// Local files/folder have priority
 	if _, err := os.Stat(name); err == nil {
-		return repo.ReadFileSystemPackage(name)
+		b, err := repo.NewBundle(name)
+		if err != nil {
+			return nil, err
+		}
+		return b.GetCRDs()
 	}
 
+	// TODO: Below is repository functionality.   All we want is a bundle!
 	// Construct the package name and download the package from the remote repo
```

Comment:
```suggestion
	// Construct the package name and download the index file from the remote repo
```
it is not actually downloading a package